charts/ocis/templates/proxy/deployment.yaml

```diff
@@ -125,6 +125,17 @@ spec:
               value: {{ .Values.features.externalUserManagement.autoprovisionAccounts.claimUserName | quote }}
             {{- end }}
 
+            {{- if .Values.features.externalUserManagement.oidc.spaceAssignment.enabled }}
+            - name: OCIS_CLAIM_MANAGED_SPACES_ENABLED
+              value: {{ .Values.features.externalUserManagement.oidc.spaceAssignment.enabled | quote }}
+            - name: OCIS_CLAIM_MANAGED_SPACES_CLAIMNAME
+              value: {{ .Values.features.externalUserManagement.oidc.spaceAssignment.claim | quote }}
+            - name: OCIS_CLAIM_MANAGED_SPACES_REGEXP
+              value: {{ .Values.features.externalUserManagement.oidc.spaceAssignment.regexp | quote }}
+            - name: OCIS_CLAIM_MANAGED_SPACES_MAPPING
+              value: {{ .Values.features.externalUserManagement.oidc.spaceAssignment.mapping | quote }}
```

Comment:
is this really working? To me it looks like it's missing a join by `,`

When I templated it, I have:

```
            - name: OCIS_CLAIM_MANAGED_SPACES_MAPPING
              value: "[oidcRoleA:manager oidcRoleB:editor]"
```